optionals/compat_ace_dispersion/script_component.hpp

```diff
@@ -0,0 +1,6 @@
+#define COMPONENT compat_ace_dispersion
+#define COMPONENT_BEAUTIFIED ACE3 Dispersion Compatibility
```

Comment:
It's not really compatibility, more that it provides realistic values.

Reply:
Same.